state/cleanup.go

```diff
@@ -147,7 +151,10 @@ func (st *State) Cleanup() (err error) {
 			err = errors.Errorf("unknown cleanup kind %q", doc.Kind)
 		}
 		if err != nil {
-			logger.Errorf("cleanup failed for %v(%q): %v", doc.Kind, doc.Prefix, err)
+			logger.Errorf(
+				"cleanup failed for %v(%q) in model %q: %v",
```

Comment:
does it read better this way or
cleanup failed in model %q for %v(%q) ?

I also don't think we need %q for the model UK is, and I'm not sure about the ellipsis. together they add 5 chars on something that is only 6 chars long

Ideally we would be looking at the strings and making sure it is reasonably understandable. have you tried just printing out the help strings and seeing what you thought?

I'm happy to be overruled as long as we have consensus.

Reply:
Fine with me; I'll make those changes.